### DISCUSSION TITLE
Awesome CRXJS

### DISCUSSION THREAD

**Discussion by jacksteamdev** [crxjs/chrome-extension-tools#81] [Show and tell]
## Are you are using CRXJS?

We're excited to see what you are making! :smiley: 

IF you are using it for a side project,
OR you are using it in your company,
AND you want to share,

:pray: Please post it here! :pray: 

Thanks in advance! :blue_heart:

**Comment by farazshuja** [comment]
Hi,

I have just created a starter template for vuejs using crxjs and like to share so others can also contribute and extend it. Along with crxjs I also added following:

- Shadow-dom for content script components. so can use css libs without colliding with page
- Webext Bridge - Messaging in WebExtension made super easy. Out of the box.
- Windi CSS - on-demand CSS utilities
- Components auto importing
- Icons - Access to icons from any icon set directly

https://github.com/farazshuja/chrome-extension-vite-vue-starter-template

thx

**Reply by octavio1243** [reply]
I know this thread is old but do you know how to solve the problem in the latest versions of Chrome that restrict and require a specific listing of web_accessible_resources?
![image](https://github.com/user-attachments/assets/d188df27-4151-4d83-a839-ba26ec7793a3)

**Comment by JoeyDoey** [comment]
Hi all!

Working on a Vite/ CRX cross-browser framework. 
Sets you up with a Vite template w/ react + ts + tailwind/ styled components.

![Bedframe - Make, Version & Publish cross-browser extensions continously with ease](https://github.com/nyaggah/bedframe/assets/284415/d545dea4-129e-42f0-82fd-5e856c655e61)

Build for one or more browsers from one code base; publish to all extension stores (current Chrome, Firefox, Edge)

I'd love to get some dev feedback. Best way to start is with the [@bedframe/cli](https://github.com/nyaggah/bedframe/tree/main/packages/cli#installation)

![bedframe-cli](https://github.com/nyaggah/bedframe/assets/284415/512540dd-18b7-4fbe-9ebe-861722b83a97)

website: [Bedframe.dev](https://bedframe.dev)
docs: coming soon

Thanks, @jacksteamdev  for the awesome crx! 👍
